libfwupdplugin/fu-device.c

```diff
@@ -3684,6 +3723,13 @@ fu_device_write_firmware(FuDevice *self,
 		return FALSE;
 	}
 
+	/* pause the polling */
+	if (fu_device_has_internal_flag(self, FU_DEVICE_INTERNAL_AUTO_PAUSE_POLLING)) {
+		g_autoptr(FuDeviceLocker) poll_locker = fu_device_poll_locker_new(self, error);
```

Comment:
Hum, the locker will be created and released instantly.

The scope for the locker should be global to the function?

Reply:
tested by rising the scope to the whole function and it works!

Reply:
doh! will fix